SCAutolib/models/user.py

```diff
@@ -0,0 +1,153 @@
+from pathlib import Path
+from python_freeipa.client_meta import ClientMeta
+
+from SCAutolib import run, logger
+
+
+class User:
+    """
+    Generic class to represent system users.
+    """
+
+    def __init__(self, username: str, password: str, pin: str, card = None,
```

Comment:
Also, I don't see here an attribute for the user directory that would be used by card